client/README.md

```diff
@@ -1,5 +1,7 @@
 # InfluxDB Client
 
+[![GoDoc](http://img.shields.io/badge/go-documentation-blue.svg?style=flat-square)](http://godoc.org/github.com/influxdb/influxdb/client)
```

Comment:
Thanks @mat

Why this badge, as opposed to https://godoc.org/github.com/influxdb/influxdb?status.png Any particular reason?